.github/workflows/build.yml

```
@@ -0,0 +1,54 @@
+name: Build
+
+on: push
+
+jobs:
+  windows_build:
+    name: Windows Build
+    runs-on: windows-latest
+    env:
+      BUILD_TYPE: Release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          ref: ${{ github.event.pull_request.head.sha }} # Check out the code of the PR head
+      - name: Compile
+        run: |
+          mkdir build
+          cmake -S . -B build -DBUILD_EXAMPLES=OFF -DBUILD_TESTING=OFF -DCAN_DRIVER=None
+          cmake --build build
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: windows-executable
+          path: |
+            ./*.dll
+            ./*.exe            
+            ./build/*.dll
+            ./build/*.exe
```

```suggestion
          path: |
            build/**/*.dll
            build/**/*.exe
```

This should do the job for the 'not found' issue, see [this run](https://github.com/Open-Agriculture/AgIsoDDOPGenerator/actions/runs/5919655233) for the result. Though I'd argue it might be better to use `build/Debug/*.dll` or `build/Release/*.dll` here instead of the folder wildcard.

---

Ah, yeah perfect! `build/Release/*.dll` and `exe` sound like the way to go to avoid grabbing duplicate copies of `SDL.dll` and stuff in `CMakeFiles`

---

Okay, well, it didn't work even with the release type set to "Release", but I think it's because we do a `set(CMAKE_BUILD_TYPE Debug)` in the main stack, which seems bad.

Opened a PR on the main stack. https://github.com/Open-Agriculture/AgIsoStack-plus-plus/pull/306